packages/backend/src/apps/gathersg/common/constants.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Loose regex to just accept only alphanumeric characters and dashes
+ * since there is no proper public documentation with GatherSG.
+ * Assumption is that the case uuid is alphanumeric and 22 characters long.
+ */
+export const CASE_UUID_REGEX = /^[a-zA-Z0-9]+$/
```

Comment:
```suggestion
export const CASE_UUID_REGEX = /^[a-zA-Z0-9]{22}$/
```

Reply:
this seems pretty confirmed right i guess?